### DISCUSSION TITLE
Cannot get response from groq when using attachments (?)

### DISCUSSION THREAD

**Discussion by stdedos** [danny-avila/LibreChat#2889] [Troubleshooting] [answered]
I have tried to start a groq/llama3-70b-8192 discussion with two images, and some text:

```
Slides #9,10 of the presenetation. Write me some presenter notes.
Enough to guide me, but not railroad me

Context is: Presenting renovatebot
```

I received:

```
chat-meilisearch  | 2024-05-28T09:12:16.824208Z  INFO index_scheduler: A batch of tasks was successfully completed with 1 successful tasks and 0 failed tasks.
LibreChat         | 2024-05-28 09:12:17 warn: [OpenAIClient.chatCompletion][stream] API error
LibreChat         | 2024-05-28 09:12:17 error: [handleAbortError] AI response error; aborting request: 400 message[0].content must be a string
chat-meilisearch  | 2024-05-28T09:12:17.634015Z  INFO HTTP request{method=POST host="meilisearch:7700" route=/indexes/messages/documents query_parameters= user_agent=node status_code=202}: meilisearch: close time.busy=5.82ms time.idle=2.59ms
```

and

![image](https://github.com/danny-avila/LibreChat/assets/133706/9f990bd5-bea8-46e3-8c92-8846a60d9482)

Just POSTing the text works

Any ideas?

**Comment by stdedos** [comment]
I think this was a bit too trigger-happy for me.

Probably, no available endpoint supports images (https://console.groq.com/docs/models).

_Ofc, it'd be nice if https://github.com/danny-avila/LibreChat could somehow detect that, and disable+inform that files won't be sent_

**Reply by stdedos** [reply, marked as answer]
_Confirmed by https://console.groq.com/ - there's no image (attachment of any kind) support_

**Reply by danny-avila** [reply]
We leave this up to you, as keeping up with what's capable, or simply deciding which ones you'd like to restrict should fall on the admin:

https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/file_config

**Reply by stdedos** [reply]
> simply deciding which ones you'd like to restrict should fall on the admin

yeah - ofc for that I agree. I just hoped that

> as keeping up with what's capable

"wasn't for me to keep up" 😅
